django/conf/__init__.py

```diff
@@ -105,6 +105,21 @@ def __init__(self, settings_module):
                 setattr(self, setting, setting_value)
                 self._explicit_settings.add(setting)
 
+        if self.SECRET_KEY_FILE and not self.SECRET_KEY:
```

Comment:
How it is currently implemented, if both `SECRET_KEY` and `SECRET_KEY_FILE` are set, `SECRET_KEY` wins.  Should this be handled differently?